File: 1145.py
n, fim = input().split(' ')
n = int(n);fim = int(fim)
for i in range(1, fim+1, n):
    for j in range(i, i+n):
        if j == i+n-1:
            print(j,end='\n')
        else:
            print(j,end=' ')

File: 1145.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let [n, fim] = lines.shift().split(' ');
let str = '';
n = parseInt(n); fim = parseInt(fim);
for(var i = 1; i < fim+1; i++){
    str += i.toString();
    if(i % n === 0){
        console.log(str);
        str = '';
    }
    else{
        str += ' ';
    }
}
